packages/nightingale-linegraph-track/src/nightingale-linegraph-track.js

```diff
@@ -0,0 +1,149 @@
+import ProtvistaTrack from "protvista-track";
+import * as d3 from "d3";
+import {
+  scaleLinear,
+  select,
+  line,
+  max,
+  min,
+  interpolateRainbow,
+  mouse,
+} from "d3";
+
+class NightingaleLinegraphTrack extends ProtvistaTrack {
+  connectedCallback() {
+    super.connectedCallback();
+
+    this._height = Number(this.getAttribute("height")) || 40;
+    this._yScale = scaleLinear();
+
+    if (this._data) this._createTrack();
+  }
+
+  set data(data) {
+    this._data = data;
+    this._createTrack();
+  }
+
+  _createTrack() {
+    select(this).selectAll("svg").remove();
+    this.svg = select(this)
```

Comment:
I know this is in the old component, but a lot of this is already present in protvista track, so we could just use `super._createTrack()`. However be careful to call it after the bit of code where max/min ranges are defined

Reply:
super._createTrack() calls this https://github.com/ebi-webcomponents/nightingale/blob/master/packages/protvista-track/src/protvista-track.js#L150. Since there are no features it will fail. Please correct me if I am wrong

Reply:
You're correct - I'm happy with the logic being taking out of `_refresh` so all good.